### PR TITLE
Another bad addtimer

### LINKED ISSUE #21893: Ai swarmers using Addtimer incorrectly
Ai swarmers are using the old Addtimer calls in startaction.

@RemieRichards

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #21893

### CODE REVIEW

**Comment by RemieRichards** [comment]
As for your passive aggressive commit name, my PR was made before the changes to ```addtimer()```, and as such, were not wrong when I wrote them.

**Comment by Cyberboss** [comment]
Yeah, one of the downsides to not having forced branch parity on merge. (<- Not passive agressive I swear)

**Comment by Cyberboss** [comment]
Alright I checked the entire github search for addtimer, god help us if there are still more. Fastmerge please

File: code/game/objects/items/charter.dm [hunk 1]
```diff
@@ -60,7 +60,7 @@
 	user << "Your name has been sent to your employers for approval."
 	// Autoapproves after a certain time
 	response_timer_id = addtimer(src, "rename_station", approval_time, \
```
Comment (lzimann) [inline code/game/objects/items/charter.dm]: can you just make it one line, i mean... there's a line right below that's at least double the size, and reading procs like this is a pain

**Comment by ExcessiveUseOfCobblestone** [comment]
if you want you can fix this one as well

https://github.com/tgstation/tgstation/blob/2c3b96be8a5e8c17aef29235ded7e9ffe8ce2e39/code/game/objects/items/charter.dm#L63

**Comment by Cyberboss** [comment]
THESE FUCKERS ARE SO STEALTHY

**Comment by Cyberboss** [comment]
Wait, that's the one I fixed the first time you nerd

**Comment by ExcessiveUseOfCobblestone** [comment]
sorry I'm illiterate

**Pull request merged**